scripts/checksum

```diff
@@ -15,30 +17,30 @@ fi
 
 FILE="$1"
 if [ -z "$1" ]; then
-    echo need filename
+    echo "Need filename" 2>&1
     exit 1
 fi
 
 LINE=$(egrep "^$FILE " "$DESCRIPTIONS")
 
 if [ $? -ne 0 ]; then
-    echo "Unknown file $FILE"
+    echo "Unknown file $FILE" 2>&1
     exit 1
 fi
 
 if [ ! -e "$FILE" ]; then
-    echo "File $FILE does not exist"
+    echo "File $FILE does not exist" 2>&1
     exit 1
 fi
 
 EXPECT=$(echo "$LINE" | awk -- '{print $2}')
 GOT=$(sha1sum "$FILE" | cut -d" " -f1)
 
 if [ "$EXPECT" != "$GOT" ]; then
-    echo "File $FILE failed checksum check"
+    echo "File $FILE failed checksum check" 2>&1
 
     if [ "$RM_ON_FAIL" ]; then
-        rm $FILE
+        rm "$FILE"
     fi
 
     exit 1
```

Comment:
True, I started with this one and moved it out of this PR and into #34. I will address your other comments in a few days :)